.kokoro/docs.sh

```diff
@@ -23,3 +23,11 @@ cd $(dirname $0)/..
 npm install
 
 npm run docs
+
+# Check broken links
+BIN=./node_modules/.bin
+
+npm install broken-link-checker
+npm install http-server
+$BIN/http-server -p 8080 docs/ &
```

Comment:
@kinwa91 I believe this causes an issue for a repo like this one, that doesn't have docs. Here's the fail log: https://source.cloud.google.com/results/invocations/445404dc-7471-412c-afa2-1fed950c5575/targets/cloud-devrel%2Fclient-libraries%2Fnodejs%2Fpresubmit%2Fgoogleapis%2Fnodejs-common%2Fnode8%2Fdocs/log